source/operations/install-deploy-manage/upgrade-minio-operator.rst

```diff
@@ -36,15 +39,147 @@ The following table lists the upgrade paths from previous versions of the MinIO
 
 .. _minio-k8s-upgrade-minio-operator-procedure:
 
-Upgrade MinIO Operator 4.2.3 and Later to |operator-version-stable|
+Upgrade MinIO Operator 4.5.8 and Later to |operator-version-stable|
 -------------------------------------------------------------------
 
 .. admonition:: Prerequisites
    :class: note
 
    This procedure requires the following:
 
-   - You have an existing MinIO Operator deployment running 4.2.3 or later
+   - You have an existing MinIO Operator deployment running 4.5.8 or later
+   - Your Kubernetes cluster runs 1.19.0 or later
+   - Your local host has ``kubectl`` installed and configured with access to the Kubernetes cluster
+
+This procedure upgrades the MinIO Operator from any 4.5.8 or later release to |operator-version-stable|.
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+The following changes apply for Operator v5.0.0 or later:
+
+- The Operator's Graphical User Interface now comes bundled with the same container as the Operator.
```

Comment:
Does this mean previously they were in separate containers and now they are both in one? `bundled with the same container` suggested multiple containers to me, but I don't know if that's correct.

Reply:
Removed this line.
For your knowledge, @feorlen, the GUI used to be part of the Console repo. It was moved in this release to the Operator repo.